### ISSUE TITLE
Unicode export in PDF is broken

### ISSUE THREAD

**Issue by andrejdam** [christianvoigt/argdown#213]
Hi again Christian,

Just wanted to inform you of this. When trying to export the map in PDF, Unicode/Cyrillic text is garbled. Only universal characters (such as numbers and punctuation) are shown as they should be. 

Check the included PDF for an example.
[Example Cyrillic argument.pdf](https://github.com/christianvoigt/argdown/files/6025559/Example.Cyrillic.argument.pdf)

You can't see now, but all nodes are full of Cyrillic text.

**Comment by christianvoigt** [comment]
Yes, the built-in fonts of pdfkit only have limited character set (see this [issue](https://github.com/foliojs/pdfkit/issues/470) and the [Readme](https://github.com/foliojs/pdfkit)).

The PdfExportPlugin should offer the option to use a different font, using [SVG-to-PDFKit's method](https://github.com/alafr/SVG-to-PDFKit#fonts) to do so.

**Comment by christianvoigt** [comment]
Ok, I added support for this in the upcoming v1.7.0. Here is an example of how to use this (with some arbitrary cyrillic letters that hopefully do not mean anything ;)):

```argdown
    ===
    svgToPdf:
        fonts:
            - {name: "arial", path: "./DejaVuSans.ttf"}
            - {name: "arial Bold", path: "./DejaVuSans-Bold.ttf"}
    ===

        [B]: ДҖЛ̕

        <A>: ДҖЛ̕
            - [B]
            + <C>: ДҖЛ̕
```

Basically you tell the PDF export to substitute the DejaVu font in the pdf file wherever "arial" is used in the svg file. I only tried this for the regular and bold font variants, but you should probably also add an entry for "arial Italic". Once you started defining custom fonts for the pdf, the export plugin will throw an error if it does not find a font name in this list. So you can simply iteratively add those names the plugin does not find until everything works (it is probably easiest to use `@argdown/cli` for that, but you should see these errors also in the "Argdown" output channel of VSCode).

Note that this might lead to layout errors if you substitute a font with very different glyph sizes. So ideally you should substitute Arial with "Arial Unicode" (I used a free software font in my tests, so I could add it to this repository).

Please let me know if this works for you (it might still take a few days until I have published all Argdown packages, but this feature is the last one that landed in v1.7.0).

**Comment by christianvoigt** [comment]
@andrejdam v1.7.0 is now published, it would be great if you could try this out, so that I can close this issue.

**Comment by andrejdam** [comment]
@christianvoigt , thanks for letting me know. You'll have to excuse me for a few days as I got sick with COVID and am unable to stay too long on my computer. As soon as I catch up my strength I'll try this out and confirm.

**Comment by christianvoigt** [comment]
Oh, I am so sorry to hear that and hope that you will get well soon! Thanks for letting me know.
